clients/spark/core/src/main/scala/io/treeverse/clients/ApiClient.scala

```diff
@@ -4,23 +4,27 @@ import com.google.common.cache.CacheBuilder
 import io.lakefs.clients.api
 import io.lakefs.clients.api.RetentionApi
 import io.lakefs.clients.api.ConfigApi
-import io.lakefs.clients.api.model.{
-  GarbageCollectionPrepareRequest,
-  GarbageCollectionPrepareResponse
-}
+import io.lakefs.clients.api.model.{GarbageCollectionPrepareRequest, GarbageCollectionPrepareResponse}
+import io.treeverse.clients.StorageAccessType.StorageAccessType
+import io.treeverse.clients.StorageUtils.{StorageTypeAzure, StorageTypeS3}
```

Comment:
I don't understand these lines: I thought we were in `package io.treeverse.clients`?!  (Perhaps "organize imports" in your IDE can fix this...)

Reply:
?

Reply:
optimize imports didn't do the trick :(

See answer number 2 here https://stackoverflow.com/a/67364323/11553804
> Just like every identifiers in scala, you need to import to access identifiers defined in another package or object